### PR TITLE
chore(docs): fix code highlighting

### LINKED ISSUE #32921: [internal] line highlights in docs are broken
Source: https://github.com/microsoft/playwright/blob/b01dd556975d2b41efcb71c482d996cf258eb190/docs/src/ci.md?plain=1#L418

Code pointers:

- https://github.com/microsoft/playwright/blob/3b86a9c0e44314aee07d71d50449545329466774/utils/markdown.js#L167
- https://github.com/microsoft/playwright/blob/3b86a9c0e44314aee07d71d50449545329466774/utils/markdown.js#L329

Grepping for `\{\d+-\d+\}` reveals only one usage - maybe we just remove it?

### PR DESCRIPTION
**Pull request by Skn0tt**
Closes https://github.com/microsoft/playwright/issues/32921

This is the diff when rolling to `playwright.dev` locally:
Screenshot 2024-10-02 at 14 54 42

### CODE REVIEW

File: docs/src/ci.md [hunk 1]
```diff
@@ -415,7 +415,7 @@ Large test suites can take very long to execute. By executing a preliminary test
 This will give you a faster feedback loop and slightly lower CI consumption while working on Pull Requests.
 To detect test files affected by your changeset, `--only-changed` analyses your suites' dependency graph. This is a heuristic and might miss tests, so it's important that you always run the full test suite after the preliminary test run.
 
-```yml js title=".github/workflows/playwright.yml"
+```yml js title=".github/workflows/playwright.yml" {20-23}
```
Comment (mxschmitt) [inline docs/src/ci.md:418]: which lines are you trying to highlight? the ones which are highlighted right now seem not related. Maybe 24-27 instead?

Also the color contrast ratio seems off, make sure its AAA compliant using the accessibility insights web extension in light and dark mode: 

image

File: docs/src/ci.md [hunk 2]
```diff
@@ -415,7 +415,7 @@ Large test suites can take very long to execute. By executing a preliminary test
 This will give you a faster feedback loop and slightly lower CI consumption while working on Pull Requests.
 To detect test files affected by your changeset, `--only-changed` analyses your suites' dependency graph. This is a heuristic and might miss tests, so it's important that you always run the full test suite after the preliminary test run.
 
-```yml js title=".github/workflows/playwright.yml"
+```yml js title=".github/workflows/playwright.yml" {20-22}
```
Comment (Skn0tt) [inline docs/src/ci.md:418]:
```suggestion
```yml js title=".github/workflows/playwright.yml" {24-26}
```

**Review by mxschmitt — APPROVED** [review]
